docs/topics/ref_known-issues-7-2-0.adoc

```diff
@@ -0,0 +1,43 @@
+:_template-generated: 2024-12-04
+:_mod-docs-content-type: REFERENCE
+
+[id="known-issues-7-2-0_{context}"]
+= Known issues
+
+This section provides highlighted known issues in {ProductFullName} version 7.2.0.
+
+.{ProductShortName} [CLI] analysis using the Container CLI fails on Windows
+
+{ProductShortName} [CLI] analysis using the Container CLI fails on Microsoft Windows and returns a `provider does not have dependency capability` error. This failure is caused by the existence of the `.metadata` folder located inside the folder where the analysis is run. Therefore, the workaround is to delete this folder. This issue is planned to be resolved in {ProductShortName} 7.2.1. link:https://issues.redhat.com/browse/MTA-4410[(MTA-4410)]
+
+.{ProductShortName} [CLI] analysis using the Containerless CLI fails on Windows
+
+{ProductShortName} [CLI] analysis using the Containerless CLI fails on Microsoft Windows and returns a `provider does not have dependency capability` error. This failure is caused by the existence of the `.metadata` folder located inside the folder where the analysis is run. Therefore, the workaround is to delete this folder. This issue is planned to be resolved in {ProductShortName} 7.2.1. link:https://issues.redhat.com/browse/MTA-4416[(MTA-4416)]
+
+.{ProductShortName} CLI analysis is extremely slow on Windows
+
+{ProductShortName} CLI analysis does not function as expected on Microsoft Windows, with analysis using container runtimes being extremely slow on Windows. This issue is planned to be resolved in {ProductShortName} 7.2.1. link:https://issues.redhat.com/browse/MTA-3667[(MTA-3667)]
+
+.Analysis of the daytrader application does not show dependencies
```

Comment:
This shouldn't mention the daytrader application.

At the root of it; this issue that is linked is about how when running analysis in containerless mode the CLI will be using the version of maven that exists on the host. We have noticed that analysis with a host that has maven 3.6 installed sometimes it will fail to fetch some dependencies.

The solution is for the user to upgrade to maven 3.9